frontend/cypress/integration/common/sidecar_injection.ts

```diff
@@ -82,7 +82,8 @@ Given('a workload without a sidecar', function () {
     });
 
     // Restart the workload to ensure changes are applied.
-    cy.exec('kubectl delete pod -n default -l app=sleep')
+    cy.exec(`kubectl scale -n ${this.targetNamespace} --replicas=0 deployment/${this.targetWorkload}`);
+    cy.exec(`kubectl scale -n ${this.targetNamespace} --replicas=1 deployment/${this.targetWorkload}`);
```

Comment:
nit: Doing
```
kubectl rollout restart deployment kiali -n istio-system
kubectl rollout status deployment kiali -n istio-system
```
might be less error prone and will wait for the restart to complete. Not sure if `scale` will wait or not.

Reply:
I was mainly doing scale down and up, because the new health indicator tests are either doing scale down or up and the env is not restored to it's original state. So, this is a way to "restart" and at the same time it is making sure that there is one replica.

That said, I think `scale` is not waiting. So, I'll do a hybrid: scale to one replica and, then restart with `rollout` to make sure it restarts in case `scale` does nothing (if there is already one replica).

Reply:
It works with the hybrid with `rollout`.

Reply:
I just updated the PR.